README.md

```diff
@@ -41,34 +38,100 @@ require('apostrophe')({
+
+Clicking on the "Export Page" button will trigger a download of the export file to your local computer.
+
+<hr>
+
+### Pieces
+![Screenshot of exporting pieces using the batch method](images/piece-batch-export.png)
+
+Unlike pages, piece-types can be exported either individually from the context menu to the right of the piece, or as a batch using the context menu above the pieces list in the manager. For example, in the image above two articles have been selected for export.
+
+Clicking export will bring up the same modal that appears when exporting pages, allowing for the selection of related documents that should also be exported.
+
+<hr>
+
+### Templates
+![Screenshot of exporting templates](images/template-export.png)
+
+If you have the [Template Library Pro](https://apostrophecms.com/extensions/template-library) module installed, templates are exported by selecting one or more within the template manager and then using the context menu located to the right of the batch operations. Like with the piece-type exporter you can elect to batch export just one or multiple files depending on how many are checked.
+
+Clicking export will bring up the same modal that appears when exporting pages, allowing for the selection of related documents that also should be exported.
+
+<hr>
+
+## Importing files
+
+![Screenshot of the utilities context menu for importing in the page manager](images/page-import.png)
+Any export file, no matter the content, can be imported using the utility context menu located at the top of the content managers, typically located to the left of the button to add new content of that type. This includes the page manager, any piece manager, or the document template manager if installed.
+
+![Screenshot of the file upload modal for importing files](images/import-file-modal.png)
+
+Clicking on the "Import" menu item will bring up a modal to select the export file you wish to import. You can only select one file at a time and the selection of an additional file will replace the first. After you select the exported file and click on the import button, a progress bar will be shown and a success or failure notification when the file has been fully imported.
+
+If the file you select has documents that already exist in your project, you'll get a notification and list of the documents that would be over-written. From that list you can choose documents you don't want imported. Note, if you have any documents that were previously published and then archived, they will trigger a duplicate overwrite warning.
+
+When importing a whole set of nested pages, the child pages will be added to the database, but will not appear in the page manager until the parent page has been imported. The parent page does not have to be imported first. Once it has been imported, any child pages that have already been imported will appear in the page manager.
```

Comment:
This feature doesn't exist yet, yeah? You can only export one page at a time right now.

Reply:
Reworded.